dev/x86_64/src/poly_use_hint_32_avx2.c

```diff
@@ -28,7 +28,7 @@
 #include "arith_native_x86_64.h"
 #include "consts.h"
 
-#define _mm256_blendv_epi32(a, b, mask)                        \
+#define mld_mm256_blendv_epi32(a, b, mask)                     \
```

Comment:
Let's rename this to MLD_MM256_BLENDV_EPI32 to make it clearer that this is actually a custom-defined macro rather than an x86 intrinsic. This tends to confuse me every time.